apps/teams-test-app/src/components/DialogAPIs.tsx

```diff
@@ -78,17 +58,14 @@ const DialogAPIs = (): ReactElement => {
             throw new Error('input is undefined');
           }
         },
-        submit: {
-          withPromise: async (dimensions, setResult) => {
+        submit: async (dimensions, setResult) => {
+          if (isTestBackCompat()) {
+            tasks.updateTask(dimensions);
+          } else {
             dialog.update.resize(dimensions);
-            setResult('Teams client SDK call dailog.update.resize was called');
-            return '';
-          },
-          //For V1 back compatibility
-          withCallback: (taskInfo, setResult) => {
-            tasks.updateTask(taskInfo);
-            setResult('Teams client SDK call tasks.updateTask was called');
-          },
+          }
+          setResult('Teams client SDK call dailog.update.resize was called');
```

Comment:
I'm going to assume fixing this typo is hard in some way because some test somewhere is looking for the string "dailog". But if it's fixable that would be neat!

Reply:
Good catch, I think that was just an oversite on my part. I will make the necessary adjustments